assets/styles/style.scss

```diff
@@ -0,0 +1,549 @@
+    }
+  }
+
+  .tp-story {
+    .ft-padd {
+      border: 0;
+
+      .card-body {
+        padding: 20px 0;
+      }
+
+      .ctext {
+        color: $b;
+        font-size: 20px;
+        font-weight: 600;
+      }
+    }
+
+    .noo-pad {
+      border: 0;
+      margin-right: 10px;
+      padding: 20px 0;
+
+      .hlink {
+        color: $hl;
+      }
+
+      img {
+        width: 95%;
+      }
+    }
+
+    .more-stories {
+      .st-padd {
+        border: 0;
+        border-top: 1px solid $hl;
+        padding: 10px 0;
+
+        .stimg {
+          float: left;
+          margin-right: 20px;
+        }
+
+        .mrlink {
+          color: $b;
+          font-size: 16px;
+        }
+
+        p {
+          font-size: 15px;
+          font-weight: 300;
+        }
+      }
+    }
+  }
+
+  .d8 {
+    border-top: 0;
+  }
+
+  .db8 {
+    border: 0;
+    border-top: 1px solid $hl;
+
+    .dbimg {
+      float: left;
+      margin-right: 20px;
+    }
+
+    .dblinkk {
+      color: $lb;
+    }
+
+    .author {
+      color: $llb;
+      font-size: 14px;
+    }
+  }
+
+  .debate {
+    .dtop {
+      border-bottom: 3px solid $bl;
+      border-top: 3px solid $red;
+      margin-bottom: 20px;
+    }
+
+    .d8 {
+      border-top: 0;
+    }
+
+    .db8 {
+      border: 0;
+      border-top: 1px solid $hl;
+
+      .dbimg {
+        float: left;
+        margin-right: 20px;
+      }
+
+      .dblinkk {
+        color: $llb;
+      }
+
+      .author {
+        color: $b;
+        font-size: 14px;
+      }
+    }
+
+    .db88 {
+      border: 0;
+      border-top: 1px solid $llb;
+
+      .dbimg {
+        float: right;
+      }
+
+      .dblink {
+        color: $llb;
+      }
+
+      .author {
+        color: $llb;
+        font-size: 14px;
+      }
+    }
+  }
+
+  .d-flex {
+    background-color: $white;
+    border-bottom: 1px solid $llb;
+    font-size: 1.1em;
+    height: 3.5em;
+    padding-top: 10px;
+
+    .nav-link {
+      border-right: 1px solid $llb;
+      color: $lb;
+    }
+  }
+
+  .row h3 {
+    color: $hl;
+    font-size: 16px;
+  }
+
+  .sgnup {
+    background-color: #f4f5f6;
+    border: 1px solid #e5e5e5;
+    color: $lb;
+    margin: 30px;
+    padding: 20px;
+
+    .top {
+      display: inline;
+
+      .logoimg {
+        float: left;
+        margin-right: 20px;
+      }
+    }
+
+    form {
+      margin-top: 15px;
+
+      .f-text {
+        border: 1px solid #e0e1e1;
+        font-weight: 300;
+        line-height: 30px;
+        padding-left: 10px;
+        width: 100%;
+      }
+
+      .submit {
+        background-color: $red;
+        border: none;
+        width: 100%;
+        margin-top: 10px;
+        height: 30px;
+        color: $white;
+      }
+    }
+  }
+
+  .magazine {
+    margin: 30px 20px;
+
+    .feats a {
+      color: #333;
+      font-size: 20px;
+      text-align: center;
+    }
+
+    .ft-pad {
+      border: none !important;
+
+      .card-body {
+        padding: 10px 0 !important;
```

Comment:
please remove all !important keywords from your CSS, cause it is a really bad practice to override CSS with it.